src/common/Tooltip/TooltipArea.tsx

```diff
@@ -233,7 +260,8 @@ export const TooltipArea = forwardRef<any, Partial<TooltipAreaProps>>(({
     }
 
     // Get the path container element
-    let target = ref.current;
+    // Note that we are using the dummy 'full' circle for alignment
+    let target = fullCircleref.current;
```

Comment:
Hey team, looks like this broke the tooltip rendering in charts other than the new Radial chart. Setting a fallback value to `ref.current` should fix it 👍 

`let target = fullCircleref.current || ref.current;`

Reply:
Oh snap - thanks for the feedback! Will address shortly.

Reply:
Published fix!